modules/codegen-plugin/src/smithy4s/codegen/Smithy4sCodegenPlugin.scala

```diff
@@ -290,6 +316,22 @@ object Smithy4sCodegenPlugin extends AutoPlugin {
       )
     )
 
+  private def prepareResolvers: PartialFunction[Resolver, String] = {
+    case mr: MavenRepository if !mr.root.contains("repo1.maven.org") => mr.root
+  }
+
+  private def prepareInputDirs(
+      base: File,
+      sourceManaged: File
+  ): PartialFunction[File, String] = {
+    // exclude files that are under sourceManaged
```

Comment:
question: can you explain why we wanna do this? I was thinking that if these files are in `smithy4sInputDirs` and affect codegen, we'd want to import them

Reply:
it's a good question. when I saw it in the list, the first time I generated it
my brain automatically marked it as unwanted because it was under `target`

maybe the fact that they are generated and user modifications are discarded because the build tool handles those files

I can keep it no problem though

Reply:
kept it

Reply:
what files are those? the ones that set metadata for the Scala import syntax, perchance?

Reply:
```
> tree bar/target/scala-2.13/src_managed/main/smithy/
bar/target/scala-2.13/src_managed/main/smithy/
└── generated-metadata.smithy
> cat bar/target/scala-2.13/src_managed/main/smithy/generated-metadata.smithy 
$version: "2"
metadata smithy4sWildcardArgument = "_"
metadata smithy4sRenderOptics = false
```